go.mod

```diff
@@ -1,28 +1,32 @@
 module github.com/rancher/cli
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.7
+toolchain go1.23.7
 
-replace k8s.io/client-go => k8s.io/client-go v0.30.1
+replace (
+	golang.org/x/net => golang.org/x/net v0.36.0
+	k8s.io/client-go => k8s.io/client-go v0.32.2
+	k8s.io/kubernetes => k8s.io/kubernetes v1.32.2
```

Comment:
k8s v1.32 is forced by norman bump in rancher/rancher https://github.com/rancher/rancher/pull/49125

Reply:
No longer relevant. Fixed with an additional `replace`.